### PR TITLE
fixed spelling

### PR DESCRIPTION
**Pull request by kajjagtenberg**
Fixed a small spelling mistake in the documentation.

### CODE REVIEW

**Comment by ovaistariq** [comment]
Thanks for fixing this.

**Comment by ovaistariq** [comment]
There are some formatting issues could you fix that:
```
Checking formatting...
[warn] docs/objects/caching.md
```

**Comment by kajjagtenberg** [comment]
Running ```npx prettier --check .``` as stated in the workflow does not change any files, nor does it complain. Even with an adjusted ```npx prettier@2.6.2 --check .``` for the given version in the pre-commit script, it does not complain. 

I cannot reproduce the formatting issue.

File: docs/objects/caching.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Unlike traditional CDNs, though, it handles dynamic data in a way that provides
 strong guarantees around freshness of data.
 
 Tigris transparently caches the objects close to the user to provide low-latency
```
Comment (ovaistariq) [inline docs/objects/caching.md:7]: Below is what I see after running it through prettier:

```
Tigris transparently caches the objects close to the user to provide low-latency
access. The region chosen for caching the objects depends on the request pattern
from the users. Objects stored in San Jose but requested frequently from Sydney
will result in getting cached in the Sydney region. Caching is provided through
a distributed global caching layer with cache nodes deployed in regions
```

Comment (kajjagtenberg) [reply docs/objects/caching.md:7]: Which version of prettier are you using? And do you use a command that differs from the one stated in the workflow?

Comment (ovaistariq) [reply docs/objects/caching.md:7]: Here is the prettier version I am using `2.6.2`. I have prettier configured in my VSCode so I don't manually run any workflow, as VSCode takes care of running the prettier plugin to fix the file as needed.

**Pull request closed without merging**